src/components/UI/Registration/Registration.js

```diff
@@ -7,15 +7,15 @@ const registsration = (props) => (
         <h1 className={classes.title}>Sign Up</h1>
         <hr></hr>
         <form className={classes.signup}>
-            <label>Name:</label>
+            <label htmlFor="name">Name:</label>
             <input type="text" placeholder="Your name here"></input>
```

Comment:
`htmlFor`'s value has to be the id of the element you want to label.

Alternatively, because id's (needing to be uniq) are always a headache with any component system (where everything is expected to be potentially reused, potentially more than once per page), you can make the `input` being labeled a child of the `label` tag, i.e.:
```
<label htmlFor="name">
  Name:
  <input type="text" placeholder="Your name here"></input>
</label>
```

While I'm commenting, you could be self-closing these empty tags, like `<input type="text" placeholder="Your name here" />`. Kind of a preference thing, they're equivalent\*, but I think self closing is neater.

\* small caveat, this is only totally true in JSX. Any JSX tag can self-close. In HTML, some tags can self-close and some can't...